pkg/authorization/registry/clusterpolicy/strategy.go

```diff
@@ -47,14 +47,12 @@ func (strategy) PrepareForUpdate(obj, old runtime.Object) {
 	_ = obj.(*authorizationapi.ClusterPolicy)
 }
 
-// Validate validates a new policy.
 func (strategy) Validate(ctx kapi.Context, obj runtime.Object) fielderrors.ValidationErrorList {
-	return validation.ValidateClusterPolicy(obj.(*authorizationapi.ClusterPolicy))
+	return validation.Validator.Validate(obj)
```

Comment:
Why make this change?  Most of these objects are going to get broken out and added upstream.  Why does this class require every object in the world to be validated?  Not a fan of changes here.